trieTools/zeroBalanceSystemAccountChecker/tokenBalanceGetter_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func createTokenBalanceBodyResponse(balance, error string) string {
+	return fmt.Sprintf(` {
+		"data": {
+			"tokenData": {
+				"balance": %s,
+			}
+		},
+		"error": "%s",
+			"code": "successful"
```

Comment:
optional: `code` is not formatted

Reply:
Right, it is indeed not even needed, since we do not search for this field, so I've removed it